### DISCUSSION TITLE
Url is whitelisted but still giving error of whitelisting

### DISCUSSION THREAD

**Discussion by tahirahmadin** [Web3Auth/Web3Auth#1121] [Help]
I'm successfully able to integrate Web3Auth in my DApp and working flawlessly in localhost.
But once I deployed the code on server it gives whitelisting error even my url is whitelisted before.

Is there any way to fix this?

<img width="302" alt="Screenshot at Dec 22 15-34-42" src="https://user-images.githubusercontent.com/25260610/209110067-36b2661a-7536-4307-9ac0-0ae7eafaa194.png">

<img width="207" alt="Screenshot at Dec 22 15-32-55" src="https://user-images.githubusercontent.com/25260610/209110035-567d4608-b9d8-4980-a339-2921ecdac076.png">

**Comment by ieow** [comment]
It seems like you are configuring for web3auth default mainnet.
When you initialize Web3auth in code, by default mainnet network is used which is mismatching with the testnet network of your Plug&play project
Please check this discussion 
https://github.com/Web3Auth/web3auth-web/discussions/897

**Reply by tahirahmadin** [reply]
<img width="389" alt="Screenshot at Dec 22 15-46-06" src="https://user-images.githubusercontent.com/25260610/209112278-4a2944b1-8d65-484a-b6b8-e13cde45cc66.png">

I've set mainnet configurations in code as well as using the clientId of mainnet and whitelist url is also added.
Still showing the error.

**Reply by ieow** [reply]
Could you try create `Mainnet` which is at the bottom selection
Or you could configure the network referring this document
https://web3auth.io/docs/sdk/flutter/initialize#arguments

**Comment by amatevosyan-github** [comment]
Seems like there is a problem with your configuration. That's how it looks for me (Web SDK) Could you provide a screenshot of yours? 
![image](https://user-images.githubusercontent.com/54032374/209114092-fd43f9af-a0a7-4781-8c1b-5a6e221da03c.png)

**Comment by tahirahmadin** [comment]
![Uploading Screenshot at Dec 22 15-56-17.png…]()

**Reply by amatevosyan-github** [reply]
Try to get another rpc target from alchemy, for example. It is free

**Reply by amatevosyan-github** [reply]
![image](https://user-images.githubusercontent.com/54032374/209121356-93d1aadc-ecfd-47dd-9c66-a515cb2c2aec.png)
Try selecting mainnet instead of aqua
